src/models/user-game-list-response.model.ts

```diff
@@ -0,0 +1,26 @@
+
+export interface GameListResponseGame {
```

Comment:
Generally I try to document every property we know something about so IntelliSense is the best it can be. Check _trophy.model.ts_ for an example.